### DISCUSSION TITLE
Failed to upload via Aspera

### DISCUSSION THREAD

**Discussion by hcutler99** [PRIDE-Archive/pride-resources#43] [PRIDE Submission Issues]
Dear PRIDE team,

Having some issues with the submission step using the PRIDE submission tool (2.8.0) and wondered whether you might be able to help me.

Everything seems to progress normally but then I receive the following error "Failed to upload via Aspera: Session error". I have tried switching to the FTP protocol, however, this doesn't get past the 'preparing submission' stage, even after a whole day. 

Do you have any ideas for things that might help?

Thanks and let me know if there is any other information I can provide to help.

Kind regards,
Harry

![Screenshot 2024-09-16 070506](https://github.com/user-attachments/assets/1217b742-80c0-4e32-9d0c-24414a79143a)

**Comment by chakrabandla** [comment]
Hi @hcutler99 , 
Sorry for the convenience caused. Can you try to submit your data using Globus. 
Here is our documentation page: https://www.ebi.ac.uk/pride/markdownpage/globus
